botocore/loaders.py

```diff
@@ -434,28 +434,42 @@ def _find_extras(self, service_name, type_name, api_version):
                 pass
 
     @instance_cache
+    def load_data_with_path(self, name):
+        """Same as ``load_data`` but returns file path as second return value.
+
+        :type name: str
+        :param name: The data path, i.e ``ec2/2015-03-01/service-2``.
+
+        :return: Tuple of the loaded data and the path to the data file
+            where the data was loaded from. If no data could be found then a
+            DataNotFoundError is raised.
```

Comment:
i don't think the second sentence is necessarily needed here.

Reply:
I decided to duplicate this from the `load_data` docstring for the convenience of the reader. It's not clear which one of the two is the "primary" function now given that `load_data` is the old one that is referenced everywhere else but `load_data_with_path` actually houses the functionality now. So someone looking for the docs could land in either place.